main.py

```diff
@@ -24,16 +35,22 @@
     print(cel_to_fahren(args.celsius))
 else:
     answer = str(input("What temperature you want convert?: ")).lower()
-
+    
     while answer not in (CELSIUM_INPUT + FAH_INPUT):
         answer = str(input("What temperature you want convert?: ")).lower()
         if answer in QUIT_INPUT:
             break
```

Comment:
@MariaMaximova, would it be better not just `break` but exit app completely? If you just break here, there might be code running after, but user chooses to exit from the app.

Reply:
Resolved in this [commit](https://github.com/MariaMaximova/convert_temp/pull/9/commits/ea07c0df2d274ab51fb7e4346d05465b52c56248)